src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -54,17 +49,23 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_TargetFrameworkVersionWithoutV>$(TargetFrameworkVersion.TrimStart('vV'))</_TargetFrameworkVersionWithoutV>
   </PropertyGroup>
 
+  <!-- Check if the Target Framework is coreclr based -->
+  <PropertyGroup Condition="'$(_IsNETCoreOrNETStandard)' == ''">
+    <_IsNETCoreOrNETStandard Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">true</_IsNETCoreOrNETStandard>
+    <_IsNETCoreOrNETStandard Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard'">true</_IsNETCoreOrNETStandard>
+  </PropertyGroup>
+
+  <!-- Workaround: https://github.com/microsoft/msbuild/issues/4948 -->
+  <PropertyGroup Condition="'$(_IsNETCoreOrNETStandard)' == 'true'">
```

Comment:
Should I've used `$(MSBuildRuntimeType)` instead of `$(_IsNETCoreOrNETStandard)`?